ServerCore/Pages/Teams/CreateTeam.razor

```diff
@@ -1,81 +1,93 @@
-                        </label>
-                    </div>
-                }
-                <div class="form-group">
-                    <label>
-                        Allow unsolicited applications
-                        <table>
-                            <tr>
-                                <td style="width:50px;vertical-align:top;">
-                                    <InputCheckbox @bind-Value="TeamModel.IsLookingForTeammates" />
-                                </td>
-                                <td>
-                                    <p>Checking this box will cause your team to show up in the list of teams that players who are looking for a team can request to join. If you want to manually invite your teammates instead, do not check this box. You will have the opportunity to approve all requests on the website.</p>
-                                </td>
-                            </tr>
-                        </table>
-                    </label>
-                </div>
-                @if (TeamModel.IsLookingForTeammates)
-                {
-                    <div class="form-group">
-                        <label>
-                            Bio for unsolicited applications (visible to applicants)
-                            <InputText @bind-Value="TeamModel.Bio" class="form-control" />
-                        </label>
-                    </div>
-                }
-                <input type="submit" />
-            </EditForm>
-        }
-        else
-        {
-            <p class="alert alert-danger">Registration is full. No further teams may be created at the present time.</p>
-        }
-    </div>
+<div class="row">
+    <div class="col-md-4">
+        @if (!EventFull)
+        {
+            <EditForm Model="TeamModel" OnValidSubmit="OnSubmit">
+                <DataAnnotationsValidator />
+                <ValidationSummary />
+                <input type="hidden" @bind="TeamModel.EventId" />
+                <div class="form-group">
+                    <label>
+                        Team name
+                        <InputText @bind-Value="TeamModel.Name" class="form-control" />
+                    </label>
+                </div>
+                <div class="form-group">
+                    <label>
+                        Team will be playing remotely
+                        <table>
+                            <tr>
+                                <td style="width:50px;vertical-align:top;">
+                                    <InputCheckbox @bind-Value="TeamModel.IsRemoteTeam" />
+                                </td>
+                            </tr>
+                        </table>
+                    </label>
+                </div>
+                @* TODO: Handle puzzleday room assignment and check hunt/safari requirements *@
+                @if (!Event.IsRemote)
+                {
+                    @if (!(Event.IsInternEvent && EventRole == ModelBases.EventRole.play))

```

Comment:
Nit: This asks remote teams for their room

Reply:
Change made in bug fix CR